packages/contracts/contracts/base/SemaphoreVerifierKeyPts.sol

```diff
@@ -0,0 +1,989 @@
+// SPDX-License-Identifier: MIT
+// Part of this file was generated with [snarkJS](https://github.com/iden3/snarkjs).
+pragma solidity 0.8.23;
+
+library SemaphoreVerifierKeyPts {
+    // Verification Key points.
+    // These values are taken from the verification key json file generated with snarkjs.
+    // It allows to use the same verifier to verify proofs for all the tree depths supported by Semaphore.
+    /*
+    uint256[14][MAX_DEPTH] VK_POINTS = [
```

Comment:
I think we can remove this comment (the list of points as uints).

Reply:
updated